suzieq/sqobjects/device.py

```diff
@@ -9,6 +9,7 @@ def __init__(self, **kwargs):
         self._valid_get_args = ['namespace', 'hostname', 'columns', 'os',
                                 'vendor', 'model', 'status', 'version',
                                 'query_str']
+        self._unique_def_column = ['model']
```

Comment:
I'm torn between model and os here. Lets leave it at what you have for now.

Reply:
Yes I thought about os too but using os we'd get only the OS name without the version, i.e.:
```
          os
0    cumulus
1        eos
2        ios
3  junos-qfx
4      linux
5       nxos
```
If we then want to use os, I think it would be more useful to get both os and (os) version, i.e.:
```
          os
0    cumulus 3.7.9
1    cumulus 4.3.0
2        eos 4.23.8M
...
```
I'll leave `model` for the moment.